docs/cli-commands.md

```diff
@@ -17,22 +17,32 @@ $ cdktf --help
 Help output:
 
 ```
-cdktf [command]
-
 Commands:
-  cdktf deploy [OPTIONS]   Deploy the given stack
-  cdktf destroy [OPTIONS]  Destroy the given stack
-  cdktf diff [OPTIONS]     Perform a diff (terraform plan) for the given stack
-  cdktf get [OPTIONS]      Generate CDK Constructs for Terraform providers and modules.
-  cdktf init [OPTIONS]     Create a new cdktf project from a template.
-  cdktf login              Retrieves an API token to connect to Terraform Cloud.
-  cdktf synth [OPTIONS]    Synthesizes Terraform code for the given app in a directory.                                                                                                    [aliases: synthesize]
+  cdktf convert [OPTIONS]                        Converts a single file of HCL configuration to Terraform
+                                                 CDK. Takes the file to be converted on stdin.
+  cdktf deploy [stack] [OPTIONS]                 Deploy the given stack                      [aliases: apply]
+  cdktf destroy [stack] [OPTIONS]                Destroy the given stack
+  cdktf diff [stack] [OPTIONS]                   Perform a diff (terraform plan) for the given stack
+                                                                                              [aliases: plan]
+  cdktf get [OPTIONS]                            Generate CDK Constructs for Terraform providers and modules.
+  cdktf import [OPTIONS] <source> <destination>  Takes the Terraform project and converts it to a CDKTF
```

Comment:
I think we talked about this previously, but this shouldn't be called `import` since `import` is used in core to import state and that would be confusing

Reply:
Thanks for pointing this out, I forgot to update the docs here :)